link/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt

```diff
@@ -55,6 +55,20 @@ internal class DefaultLinkAccountManager @Inject constructor(
                     lookup = consumerSessionLookup,
                     startSession = startSession,
                 )
+            }.mapCatching { account ->
```

Comment:
similar to [this comment](https://github.com/stripe/stripe-android/pull/9478#discussion_r1811051749) -- this will impact inline sign up and we don't want that, right?